### DISCUSSION TITLE
So many updates

### DISCUSSION THREAD

**Discussion by marcusleonas** [JanDeDobbeleer/oh-my-posh#5950] [General]
I have autoupdate on and it seems that multiple times a day there are updates being released, why? can't all of these updates be saved and you guys do a weekly update? it's really annoying having to wait for 30s every time i boot my shell just to stay up to date,

**Comment by marcusleonas** [comment]
Like the 24.10.1 update literally removes like 5 lines of code why does it need to have its own update?

**Reply by JanDeDobbeleer** [reply]
Because that's how it works, code gets changed, things get merged and released. An auto update interval might be a good idea, no?

**Reply by JanDeDobbeleer** [reply]
@marcusleonas good news, you can now tweak the behaviour. See [here](https://ohmyposh.dev/docs/installation/upgrade).
